treescript/src/treescript/gecko/versionmanip.py

```diff
@@ -31,9 +31,7 @@
 }
 
 _VERSION_CLASS_PER_END_OF_SOURCE_REPO = {
-    "fenix": MobileVersion,
-    "firefox-android": MobileVersion,
-    "focus-android": MobileVersion,
+    "placeholder-repo": MobileVersion,  # not currently used
```

Comment:
Any reason to not just remove the unused code?

Reply:
Sorry, I'm not sure I understand what you had in mind. I removed that `placeholder-repo` line - it didn't add value. Were you thinking of removing `_VERSION_CLASS_PER_END_OF_SOURCE_REPO`? I've left it thinking that it's generic and might be useful again.
Of course `treescript/github` is also entirely unused now, but I think that's useful in the foreseeable future. Removing that, or cleaning up the github tests further, seems to get a little complicated; I wouldn't mind leaving that for a follow-up.